packages/app/app/i18n.js

```diff
@@ -32,7 +32,7 @@ const languageDetector = {
 export const setupI18n = () => {
   return i18n.use(languageDetector).init({
     fallbackLng: 'en',
-    debug: process.env.NODE_ENV !== 'production',
```

Comment:
always false?

Reply:
i disabled this because the logs where annoying, i forgot to put it back. not sure if it usefull

Reply:
yeah the logs are annoying most of the time, but maybe sometimes it'd be useful so that users can provide more debugging info. I'll think about how to do that in the future, let's keep them off for now and enable them only when we need, locally...